openssl-dynamic/src/main/c/jnilib.c

```diff
@@ -98,7 +102,15 @@ JNIEXPORT jint JNICALL JNI_OnLoad_netty_tcnative(JavaVM *vm, void *reserved)
     tcn_parent_pid = getppid();
 #endif
 
-    return  TCN_JNI_VERSION;  
+    TCN_LOAD_CLASS(env, byteArrayClass, "[B", JNI_ERR);
```

Comment:
Thats done in the macro

> Am 22.08.2016 um 16:58 schrieb Nathan Mittler <notifications@github.com>:
> 
> In openssl-dynamic/src/main/c/jnilib.c:
> 
> > @@ -98,7 +102,15 @@ JNIEXPORT jint JNICALL JNI_OnLoad_netty_tcnative(JavaVM *vm, void *reserved)
> >      tcn_parent_pid = getppid();
> >  #endif
> >  
> > -    return  TCN_JNI_VERSION;  
> > +    TCN_LOAD_CLASS(env, byteArrayClass, "[B", JNI_ERR);
> do you need to convert to global references?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>